### ISSUE TITLE
idea: multi-stage docker build

### ISSUE THREAD

**Issue by ka7** [alphacep/vosk-server#140]
https://docs.docker.com/develop/develop-images/multistage-build/

there is a way, to have multiple stages for docker-build. e.g. 
1st to build, require gcc
2nd to run, needs just the binary from (1)

with that, the docker-files would become smaller. 
who needs the g++, make, cmake, ...  later within the docker-image ?

**Comment by nshmyrev** [comment]
Yeah, we could actually just install the pip package. Originally we wanted to provide some flexibility for the build (add an option for mkl, etc). We might reconsider that later and go with simple pip3 install vosk in the docker.
